docs/cost/ocds.md

```diff
@@ -0,0 +1,150 @@
+# OCDS to OC4IDS Mapping
+
+<style>
+.wy-nav-content {
+  max-width: 1200px;
+}
+</style>
+
+## Guidance
+
+## Command-line tool and reference implementation
```

Comment:
This file also has most of the indentation incorrect.

Reply:
Oh dear. That'll teach me to use find and replace to update heading levels! Fixed in https://github.com/open-contracting/infrastructure/pull/422/commits/4df60d29b313b457a3b943f510c7901edda41806

Reply:
FWIW I tend to replace `^##` with `#` (can also do `#(#+)` with `$1`) to reduce the heading level (syntax might differ across editors).

Reply:
Nice. Thanks!